python/cog.py

```diff
@@ -31,6 +32,11 @@
 fh.setFormatter(formatter)
 logger.addHandler(fh)
 
+EXP_TEMPLATE_SPLIT_IDX = 9
+export_template = []
+with open("python/export_template.py") as f:
```

Comment:
I think the best thing to do in this case is to add a module in python/ that contains the pieces of the template as strings. It can do all the generation and just return the contents as a string. The calling function should handle ownership of any files it needs to create. I'll refactor this.